corehq/apps/domain/views.py

```diff
@@ -2809,3 +2813,18 @@ def post(self, request, domain):
             return self._generic_error()
 
         return json_response({'cards': self.payment_method.all_cards_serialized(self.account)})
+
+
+class PasswordResetView(View):
+    urlname = "password_reset_confirm"
+
+    def get(self, request, *args, **kwargs):
+        return password_reset_confirm(request, *args, **kwargs)
+
+    def post(self, request, *args, **kwargs):
+        response = password_reset_confirm(request, *args, **kwargs)
+        uidb64 = kwargs.get('uidb64')
+        uid = urlsafe_base64_decode(uidb64)
+        user = User.objects.get(pk=uid)
+        clear_failed_logins_and_unlock_account(None, request, user)
```

Comment:
nit: since you are calling this not as a signal, maybe you can make this a function somewhere else that the signal also calls, that way the first argument isn't a weird `None`.